### ISSUE TITLE
path is undefined

### ISSUE THREAD

**Issue by raytio** [ownstats/ownstats#6]
The generated `hello.js` contains the following:
```
          (c.src =
            "https://d3dfiqfbrgk5v0.cloudfront.net/undefined?" + h.join("&"));
```

It looks like that's because `path` from `TrackingScriptGenerator.js` is undefined so `var reqUrl = 'https://' + hostname + path;` is evaluating to `https://d3dfiqfbrgk5v0.cloudfront.net/undefined`

What is the path used for? Is that expected behaviour?

**Comment by tobilg** [comment]
Path should be „/hello.gif“ in that case. Will have a look tomorrow.

**Comment by tobilg** [comment]
This was fixed in 42d75aec63ac00364ba2f74da2fc72c0961ad217
